helpdesk_mgmt/views/helpdesk_ticket_templates.xml

```diff
@@ -20,7 +20,7 @@
               </t>
               <form method="POST" t-attf-action="/new/ticket">
                 <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
-                <button name="create_new_ticket" type="action" class="btn btn-primary" groups="base.group_portal" style="float: right; margin-right: 0px; margin-top:5px;">New Ticket</button>
```

Comment:
Instead of removing this, you should add both `base.group_portal` and `base.group_user`. If not, public website users will be able to submit a ticket.

Reply:
@pedrobaeza comment addressed!